### PR TITLE
Jhalackova micro cloud

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

SLE Miro for public cloud.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### CODE REVIEW

**Comment by lvicoun** [comment]
/home/jana/SUSE/DOCS/doc-modular/build/SLE-Micro-public-cloud/SLE-Micro-public-cloud_en.pdf

File: xml/concept-micro-cloud.xml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    <para>
+      The instance intialization is performed using &ignition; and Afterburn.
+      In the default &ignition; configuration, &ignition; is responsible for
+      creating the default user: <literal>suse</literal>. &ignition; interacts
+      with the Cloud provider instance metadata service only to get so called
+      user data. Other data, like the user specified in Azure or SSH key from
+      the GCE, are not parsed from this metadata service
+    </para>
+    <para>
+      Afterburn passes the SSH key to the <literal>suse</literal> home
```
Comment (rjschwei) [inline xml/concept-micro-cloud.xml:44]: substitute "passes" with "writes"

**Review by rjschwei — APPROVED** [review]

File: xml/SLE_Micro.asm.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+     <!-- ISO date of last update as YYYY-MM-DD -->
+     <meta name="updated" content="2023-03-09"/>
+     <!-- enter the platform identifier or list of comma-separated list of identifiers -->
+     <meta name="architecture" content="x86_64;zseries;power;aarch64"/>
+     <!-- enter product name and version -->
+     <meta name="productname">
+       <productname version="SLEMICRO">&slem;</productname>
+      </meta>
+     <!-- SEO and social media -->
+     <meta name="title">SLE Micro in public clouds</meta>
```
Comment (dariavladykina) [inline xml/SLE_Micro.asm.xml:67]:
```suggestion
     <meta name="title">SLE Micro in Public Clouds</meta>
```

**Review by dariavladykina — COMMENTED** [review]
Hi Jana, please see some suggestions here. We could also do another review to see how the titles and entities are rendered. Thanks!

File: xml/task-slemicro-useradd.xml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    <procedure>
+      <step>
+        <para>
+          Run the <command>useradd</command> command as follows:
+        </para>
+<screen>&prompt.root;<command>useradd --btrfs-subvolume-home --create-home <replaceable>USER_NAME</replaceable></command></screen>
+        <para>
+          The <option>--btrfs-subvolume-home</option> denotes that the
+          <filename>/home</filename> directory is mounted as a subvolume. The
+          <option>--create-home</option> creates the <filename>/home</filename>
```
Comment (lvicoun) [inline xml/task-slemicro-useradd.xml:50]:
```suggestion
          <option>--create-home</option> creates the <filename>/home</filename> directory
```

**Pull request merged**